modules/gdnative/gdnative/string.cpp

```diff
@@ -399,7 +399,21 @@ godot_string GDAPI godot_string_num_int64(int64_t p_num, godot_int p_base) {
 
 godot_string GDAPI godot_string_num_int64_capitalized(int64_t p_num, godot_int p_base, godot_bool p_capitalize_hex) {
 	godot_string result;
-	memnew_placement(&result, String(String::num_int64(p_num, p_base, true)));
+	memnew_placement(&result, String(String::num_int64(p_num, p_base, p_capitalize_hex)));
```

Comment:
It made no sense to have a `p_capitalize_hex` argument but always use `true`.
This could have been fixed by simply removing the bool, but that would break the API so instead I used it.